packages/cli/src/commands/capabilities.ts

```diff
@@ -164,6 +164,15 @@ export async function getStandard(client: SmartThingsClient): Promise<Capability
 	return caps.map((capability: CapabilitySummary) => { return { ...capability, namespace: 'st' } })
 }
 
+export async function getAllFiltered(client: SmartThingsClient, filter: string): Promise<CapabilitySummaryWithNamespace[]> {
+	const list = (await Promise.all([getStandard(client), getCustomByNamespace(client)])).flat()
+	if (filter) {
+		filter = filter.toLowerCase()
+		return list.filter(it => it.id.toLowerCase().includes(filter) && it.status !== 'deprecated')
+	}
+	return list
```

Comment:
Should we filter out deprecated ones here too?

Reply:
We allow deprecated capabilities to be used if explicitly entered, but it seems to me like we should not be prompting the user to add them, since we don't really want them to be used in new devices